test/index.js

```diff
@@ -0,0 +1,18 @@
+var R = require('..');
+var fs = require('fs');
+var path = require('path');
+var eq = require('./shared/eq');
+
+
+describe('api surface', function() {
+  var srcFolder = path.join(__dirname, '../src');
+  var isJsFile = function(file) { return file.match(/\.js$/); }
+  var removeJsEnding = function(file) { return file.replace('.js', ''); }
+
+  var exportedApi = Object.keys(R);
+  var actualApi = fs.readdirSync(srcFolder).filter(isJsFile).map(removeJsEnding);
```

Comment:
other side note is: `npm test` also runs `npm run build`, which in turns run `make`.

Every time you want to test your code, you need to run build, which in turns ruin fast tests. its kinda said to me